Tests/RevenueCatUITests/Data/__Snapshots__/PaywallDataValidationTests/testValidateMissingPaywallWithSpanishLocalization.1.json

```diff
@@ -0,0 +1,44 @@
+{
+  "asset_base_url" : "https://assets.pawwalls.com",
+  "config" : {
+    "blurred_background_image" : true,
+    "colors" : {
+      "light" : {
+        "accent1" : "#FFFFFF",
+        "accent2" : "#FFFFFF",
+        "background" : "#FFFFFF",
+        "call_to_action_background" : "#FFFFFF",
+        "call_to_action_foreground" : "#FFFFFF",
+        "text1" : "#FFFFFF"
+      }
+    },
+    "display_restore_purchases" : true,
+    "images" : {
+      "background" : "background.jpg",
+      "header" : null,
+      "icon" : "revenuecatui_default_paywall_app_icon"
+    },
+    "packages" : [
+      "$rc_monthly",
+      "$rc_annual"
+    ],
+    "privacy_url" : null,
+    "tos_url" : null
+  },
+  "localized_strings" : {
+    "en_US" : {
+      "call_to_action" : "Continuar",
+      "call_to_action_with_intro_offer" : null,
+      "features" : [
+
+      ],
+      "offer_details" : "{{ total_price_and_per_month }}",
+      "offer_details_with_intro_offer" : "Comience su prueba de {{ sub_offer_duration }} y luego {{ total_price_and_per_month }}.",
```

Comment:
Wonder if these would look more natural as a `tu` than a `usted`, I imagine most apps don't use `usted`. 
Then again I should check, I've no idea 😂

Reply:
How about infinitive (like Continuar already is)?
> Comenzar prueba de {{ sub_offer_duration }} y luego {{ total_price_and_per_month }}.

Reply:
I'll change this to the default from StoreKit:
![image](https://github.com/RevenueCat/purchases-ios/assets/685609/72664653-e30f-4aeb-997d-12feb18db13a)